riseagain47/EPIC-Mail
# Ch create utility model #164522029

#### What does this PR do?
- Create utility dummy data model for App API

#### Description of Task to be completed?
- Create a utils folder and corresponding dummy data models for entities specified

#### How should this be manually tested?
- User can view the folder structure and under server folder will see a utils folder which contains JavaScript files for different dummy data for entities specified.

#### What are the relevant pivotal tracker stories?
- Chore #164522029 - Create utility model
